wasmcloud_host/lib/wasmcloud_host/actor_watcher.ex

```diff
@@ -23,9 +25,12 @@ defmodule WasmcloudHost.ActorWatcher do
   end
 
   def handle_info({:file_event, _watcher_pid, {path, events}}, state) do
-    if :modified in events and :closed in events do
-      {:ok, bytes} = File.read(path)
-      actor_id = Map.get(state, path, "")
+    # Modified is emitted on Mac, Windows, and Linux when a file is changed
+    if :modified in events do
+      {:ok, _bytes} = File.read(path)
```

Comment:
Minor nit: What is this read asserting? Is it to check that the file actually exists? I'd recommend using the equivalent of stat-ing the file rather than reading the whole module in and throwing away the bytes. If people end up doing larger modules, this would take a while every time a reload is triggered

Reply:
We actually don't need that at all, it was an artifact of refactoring. I'll remove it